users/api/endpoint.go

```diff
@@ -194,27 +227,27 @@ func removeUserFromGroup(svc users.Service) endpoint.Endpoint {
 	}
 }
 
-func listUsersForGroupEndpoint(svc users.Service) endpoint.Endpoint {
+func listMembersEndpoint(svc users.Service) endpoint.Endpoint {
```

Comment:
I like that you renamed this. I like `memberships` and `members` terminology, it's unambiguous and short.

Reply:
We agreed with @mteodor to use this naming. I'm just polishing